raiden-dapp/src/services/raiden-service.ts

```diff
@@ -49,14 +50,29 @@ export default class RaidenService {
     stateBackup?: string,
     subkey?: true,
   ): Promise<Raiden> {
+    let storageOpts:
+      | { state: string | undefined }
+      | { state: string | undefined; adapter: string; iosDatabaseLocation: string } = {
+      state: stateBackup,
+    };
+    // When running in capacitor, enable native sqlite plugin
+    if (process.env.VUE_APP_DB_ADAPTER === 'cordova-sqlite') {
+      const { default: adapterPlugin } = await import('pouchdb-adapter-cordova-sqlite');
```

Comment:
Just a headsup: this may cause webpack or other bundlers to always include this dependency, because they don´t know this codepath isn´t reachable. There should be some way to tell them to tree-shake this out depending on the platofmr/env vars at build time, etc. I got this issue in the past with the SDK, where an `await import` triggered some devDependency to be required as peer-dependency of projects using it, so may be worth it to check if there´s an easy way to `#ifdef` this out

Reply:
Is there an example for this? I cannot find anything in the codebase...

Reply:
No, this is up for research.

Reply:
I checked the `dist` sizes on master and this branch for a normal `yarn build´. They are equal, so I guess we don't need to handle this now:
```
➜  raiden-dapp git:(master) ✗ du -h dist
868K	dist/css
3.6M	dist/js
 60K	dist/img/icons
280K	dist/img
3.4M	dist/fonts
8.2M	dist




➜  raiden-dapp git:(sqlite-plugin) ✗ du -h dist
868K	dist/css
3.4M	dist/js
 60K	dist/img/icons
280K	dist/img
3.4M	dist/fonts
8.1M	dist
```